app/models/concerns/locale_matchable.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+module LocaleMatchable
+  extend ActiveSupport::Concern
+
+  included do
+    scope :matching_locale,
+          ->(locale) do
+            regionless_locale = locale.to_s.split("_").first
+            where("locale LIKE ?", "#{regionless_locale}%")
```

Comment:
Curious, what does `regionless` mean here?

Reply:
Like, `en` is the locale in `en_GB` we could define `GB` as a region (/country)